api/Element.json

```diff
@@ -300,13 +300,7 @@
             "spec_url": "https://w3c.github.io/csswg-drafts/web-animations-1/#dom-keyframeeffectoptions-composite",
             "support": {
               "chrome": {
-                "version_added": "79",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#enable-experimental-web-platform-features"
-                  }
-                ]
+                "version_added": "64"
```

Comment:
Chrome 64 is supported by https://storage.googleapis.com/chromium-find-releases-static/416.html#41658d028f33d2ce5992165844e4dad9ff2015f5, but I suspect this is yet another case of dictionary members being recognized before actually having an effect.

The same CL added support for KeyframeEffect.composite, and that's marked as Chrome 84, which I've confirmed is correct. That's when a lot of Web Animations support shipped.

I haven't found the feature flag check that makes this a no-op, but suspect it's there somewhere.

I think it'd be more conservative to say this is supported since 84, unless we have tests showing it having an effect before then.